### DISCUSSION TITLE
Associated parts - how this function work?

### DISCUSSION THREAD

**Discussion by agb2008** [Part-DB/Part-DB-server#545] [Q&A] [answered]
When I define a new part in Part-DB (I use current version 1.11.1) there is an option "Associated parts" with three different relation types:
1. Other (custom value)
2. Is compatible with
3. Supersedes
I try to set possible replacement parts for my project, so when I use the option "Build" for my project, and the main part is not available (not in stock) then I assume that the replacement part would be used... But I can't make it work, regardless of what option I set in "Associated parts"  - it would just be ignored. And system states that I can't build my project because of missing parts. Am I doing something wrong, or is it supposed to work this way? Any workaround available?

**Comment by jbtronics** [comment, marked as answer]
The parts associations are currently purely semantical / informational and trigger no other specific behavior.

So for now you can use this to mark two parts as (more or less) compatible with each other for your own information, but the project subsystem does not care for this information (yet).

For this something like metaparts would be more useful, but these are not existing (yet).

**Reply by agb2008** [reply]
Thank you, for your fast reply. I should probably file an enhancement request on this feature. It would be quite useful if this feature would be fully implemented.
